src/main/java/org/sagebionetworks/web/client/jsinterop/SRC.java

```diff
@@ -46,28 +56,63 @@ public static class SynapseComponents {
-    public static ReactComponentType<EntityActionMenuPropsJsInterop> EntityActionMenu;
+    public static ReactComponentType<
+      EntityPageBreadcrumbsProps
+    > EntityPageBreadcrumbs;
+    public static ReactComponentType<
+      EntityPageTitleBarProps
+    > EntityPageTitleBar;
+    public static ReactComponentType<
+      EntityActionMenuPropsJsInterop
+    > EntityActionMenu;
     public static ReactComponentType<HtmlPreviewProps> HtmlPreview;
-    public static ReactComponentType<CreatedByModifiedByProps> CreatedByModifiedBy;
-    public static ReactComponentType<TwoFactorAuthSettingsPanelProps> TwoFactorAuthSettingsPanel;
-    public static ReactComponentType<TwoFactorBackupCodesProps> TwoFactorBackupCodes;
-    public static ReactComponentType<TwoFactorEnrollmentFormProps> TwoFactorEnrollmentForm;
+    public static ReactComponentType<
+      CreatedByModifiedByProps
+    > CreatedByModifiedBy;
+    public static ReactComponentType<
+      TwoFactorAuthSettingsPanelProps
+    > TwoFactorAuthSettingsPanel;
+    public static ReactComponentType<
+      TwoFactorBackupCodesProps
+    > TwoFactorBackupCodes;
+    public static ReactComponentType<
+      TwoFactorEnrollmentFormProps
+    > TwoFactorEnrollmentForm;
     public static ReactComponentType<EmptyProps> SubscriptionPage;
-    public static ReactComponentType<AccessRequirementListProps> AccessRequirementList;
-    public static ReactComponentType<TableColumnSchemaEditorProps> TableColumnSchemaEditor;
+    public static ReactComponentType<
+      AccessRequirementListProps
+    > AccessRequirementList;
+    public static ReactComponentType<
+      TableColumnSchemaEditorProps
+    > TableColumnSchemaEditor;
     public static ReactComponentType<EntityHeaderTableProps> EntityHeaderTable;
-    public static ReactComponentType<AvailableEvaluationQueueListProps> AvailableEvaluationQueueList;
-    public static ReactComponentType<AccessRequirementRelatedProjectsListProps> AccessRequirementRelatedProjectsList;
-    public static ReactComponentType<CreateTableViewWizardProps> CreateTableViewWizard;
-    public static ReactComponentType<SqlDefinedTableEditorModalProps> SqlDefinedTableEditorModal;
-    public static ReactComponentType<EntityViewScopeEditorModalProps> EntityViewScopeEditorModal;
+    public static ReactComponentType<
+      AvailableEvaluationQueueListProps
+    > AvailableEvaluationQueueList;
+    public static ReactComponentType<
+      AccessRequirementRelatedProjectsListProps
+    > AccessRequirementRelatedProjectsList;
+    public static ReactComponentType<
+      CreateTableViewWizardProps
+    > CreateTableViewWizard;
+    public static ReactComponentType<
+      SqlDefinedTableEditorModalProps
+    > SqlDefinedTableEditorModal;
+    public static ReactComponentType<
+      EntityViewScopeEditorModalProps
+    > EntityViewScopeEditorModal;
+    public static ReactComponentType<
+      AccessRequirementAclEditorProps
+    > AccessRequirementAclEditor;
```

Comment:
This is the only change in this file -- the rest are formatting changes

Reply:
To be honest this is less readable...I don't know what prettier is doing